src/vs/workbench/api/common/extHostDebugService.ts

```diff
@@ -1019,7 +1020,7 @@ export class ExtHostVariableResolverService extends AbstractVariableResolverServ
 				}
 				return undefined;
 			}
-		}, undefined, Promise.resolve(process.env));
+		}, undefined, undefined, Promise.resolve(process.env));
```

Comment:
Same issue with the [remoteTerminalChannel.ts](https://github.com/microsoft/vscode/pull/141902/files#diff-5de37d9e036c25970fe775ae97663a4be7f604fa0b17ccc50e4798bbda5f4f5b)